app/views/components/_filter_section.html.erb

```diff
@@ -0,0 +1,25 @@
+<% add_app_component_stylesheet("filter-section") %>
+<%
+  raise ArgumentError, "title is required" unless local_assigns[:title]
+  shared_helper = GovukPublishingComponents::Presenters::SharedHelper.new(local_assigns)
+  open ||= nil
+  status_text ||= ""
+  title ||= ""
+
+  margin_bottom ||= 3
+  summary_aria_attributes ||= {}
+
+  component_helper = GovukPublishingComponents::Presenters::ComponentWrapperHelper.new(local_assigns)
+  component_helper.add_class("app-c-filter-section")
+  component_helper.add_class(shared_helper.get_margin_bottom)
+  component_helper.set_open(open)
+%>
+
+<%= tag.details(**component_helper.all_attributes) do %>
+  <%= tag.summary class: "app-c-filter-section__summary", aria: summary_aria_attributes do %>
+    <div class="app-c-filter-section__summary-content">
+      <%= sanitize("<span class=\"app-c-filter-section__summary-title\"><span class=\"govuk-visually-hidden\">filter by </span>#{title}</span><span class=\"app-c-filter-section__summary-status\">#{status_text}</span>") %>
```

Comment:
Don't think this needs to be output as a string. Also, should we not include the status span entirely if none is given?

```erb
<span class="app-c-filter-section__summary-title">
  <span class="govuk-visually-hidden">Filter by</span>
    <%= title %>
  </span>

  <% if status_text.present? %>
    <span class="app-c-filter-section__summary-status">
      <%= status_text %>
    </span>
  <% end %>
</span>
```

Reply:
markup was slightly wrong with closing tags, but in essence changed it as per your suggestion